build/netlify-docs.js

```diff
@@ -16,4 +17,9 @@ if (process.env.BRANCH === 'master' && semver.valid(output.trim()) === null) {
 
   // copy the legacy docs over
   sh.cp('-R', 'docs/legacy-docs', 'docs/api/docs');
+
+  if (process.env.BRANCH !== 'master') {
```

Comment:
there's no need or want to generate this in the master branch currently, I don't think.